pallets/payment/src/lib.rs

```diff
@@ -179,12 +162,12 @@ pub mod pallet {
 				amount,
 				PaymentState::Created,
 				T::IncentivePercentage::get(),
-				Some(remark),
+				remark.clone(),
```

Comment:
```suggestion
				&remark,
```
Following the orml PR discussion, I think it's going to be ok to not bother with the storage of the remark since we already publish it in the created event and only need it while creating the payment to pass it the `FeeHandler`. That means we need to add the extra argument to the `apply_fees` method to accept our `&remark`

Reply:
Still don't see the need for `clone()`. Maybe even better than borrowing the option could be to receive an `Option<&[u8]>`(`remark.as_ref().map(Borrow::borrow)`) which is more ergonomic and won't break future user's fee handler code if we decide to change the bounded vec for something else.